test/integration/smoke/test_disk_offerings.py

```diff
@@ -90,9 +90,81 @@ def test_01_create_disk_offering(self):
                         )
         return
 
+    @attr(tags=["advanced", "basic", "eip", "sg", "advancedns", "smoke"], required_hardware="false")
+    def test_02_create_disk_offering_with_cache_mode_type(self):
```

Comment:
Why this renumbering/renaming? It can break statistic analysis of test breakage or comparison. Why not add them to the end with new numbers?